engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -1227,11 +1227,21 @@ IR.Expression translateNumber(Tree.Number ast) {
   }
 
   IR.Literal translateLiteral(Tree.TextLiteral txt) throws SyntaxException {
+    if (txt.getClose() == null) {
+      if (txt.getOpen() == null || switch (txt.getOpen().codeRepr()) {
```

Comment:
When can getOpen ever be null?

Reply:
When _things go wrong_ ;-) Whether or not _things can go wrong_ in such a way is hidden somewhere in the (mostly) Rust code. Anyway I am sure we don't want `NullPointerException` but rather a proper syntax error in such situation.

Reply:
Right, I read the code wrong and thought that then it is accepted as valid literal. Now I see it is then a syntax error and that definitely sounds right!